db/migrate/20210915145551_create_alerts.rb

```diff
@@ -0,0 +1,18 @@
+class CreateAlerts < ActiveRecord::Migration[6.0]
+  def change
+    create_table :alerts do |t|
+      t.string :name
+      t.string :metric_name
+      t.integer :threshold
+      t.string :emails
```

Comment:
wouldn't it be better to use an array for emails? https://guides.rubyonrails.org/active_record_postgresql.html#array